src/pages/docs/golem/overview/golem-token.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Golem Tokens explained.
+description: Introduction to Golem Network Tokens, addresses, and networks
+type: Article
+---
+
+# What is a GLM?
+
+GLM is an ERC-20 utility token token, used as a currency to pay for renting resources from Golem Network providers.
+
+## Where to acquire GLM?
+
+### Buy
+
+You can acquire GLM on major crypto exchanges:
+
+- decentralized:
+  - [Quick Swap](https://quickswap.exchange/)
+  - [Uniswap](https://uniswap.org/)
+- and centralized platforms:
+  - [Binance](https://www.binance.com/)
+  - [Coinbase](https://www.coinbase.com/)
+  - [Crypto.com](https://crypto.com/)
+  - [OKX](https://www.okx.com/)
+  - [Gate.io](https://www.gate.io/)
+  - [KuCoin](https://www.kucoin.com/)
+  - [HTX](https://www.htx.com/)
+  - [Bithumb](https://www.bithumb.com/)
+
+{% alert level="warning" %}
+
+Note that due to transaction costs on the Ethereum mainnet, we recommend choosing the Polygon network for your payments. Therefore you should rather acquire GLM directly on the Polygon network to avoid the cost of bridging them from Ethereum.
+
+{% /alert %}
+
+Another way is to use our [Onboarding portal](https://glm.golem.network) where you will be assisted through the whole process.
+
```

Comment:
I would put this above list of CEX and DEX. We also should pinpoint that Onboarding portal is recommended way of obtaining tokens.

Reply:
I will keep the current structure.